src/document/DocumentAnnotator.ts

```diff
@@ -1,39 +1,45 @@
-import debounce from 'lodash/debounce';
 import BaseAnnotator, { Options } from '../common/BaseAnnotator';
 import BaseManager from '../common/BaseManager';
+import HighlightListener from '../highlight/HighlightListener';
 import { centerRegion, isRegion, RegionManager } from '../region';
 import { Event } from '../@types';
 import { getAnnotation } from '../store/annotations';
 import { getSelection } from './docUtil';
 import { HighlightManager } from '../highlight';
```

Comment:
Can we do `import { HighlightListener, HighlightManager } from '../highlight';` here?